src/index.html

```diff
@@ -11,7 +11,193 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+     class="form-1"
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+    >
+      <fieldset class="field">
+
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+           type="text"
+           name="Surname"
+           autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+           type="text"
+           name="Name"
+           autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+           type="number"
+           name="Age"
+           min="1"
+           max="100"
+           value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+           type="date"
+           name="Birth"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+           type="checkbox"
+           name="Accepts"
+           required
+          >
+        </label>
+      </fieldset>
+    </form>
+    <form
+     class="form-1"
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+     >
+      <fieldset class="field">
+
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+           type="email"
+           name="email"
+           placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+           type="password"
+           name="password"
+           minlength="6"
+           maxlength="16"
+          >
+        </label>
+      </fieldset>
+    </form>
+    <form
```

Comment:
```suggestion
    </form>
    
    <form
```
add empty line between same level elements